### ISSUE TITLE
Linux DGL requirement won't run on other systems

### ISSUE THREAD

**Issue by smithna** [deusebio/Graph-Machine-Learning#23]
This is OK for Docker, but won't work on in-Linux environments.

https://github.com/deusebio/Graph-Machine-Learning/blob/e6e7024ff8cec77b8bce26a0f14b871debae4c63/Chapter05/requirements.txt#L22

**Comment by smithna** [comment]
The same import is used in Chapter07.

**Comment by smithna** [comment]
Several other Linux imports are also specified in Chapter07.

**Comment by deusebio** [comment]
Yes, this is correct. Unfortunately, DGL has discontinued support on darwin and win platforms. See [here](https://www.dgl.ai/pages/start.html) where they provide a disclaimer:

> Since 2024.06.27, we have stopped providing packages for Windows and MacOS. The latest version of available package is 2.2.1.

Here we are currently using version 2.4.0. I agree that is indeed a fair point to be highlighted, in the repository but also in the text. We could leave the issue open making sure it does not fall through the cracks, in the chapters as well.

**Comment by deusebio** [comment]
The disclaimer has been added in Chapter 3, in the DGL technical discussion. Closing the issue now
